pillars/5_HST__POD/redirects__core.sls

```diff
@@ -84,6 +85,9 @@ nginx:
     - crt: redirects.creativecommons.org
       src: donate.creativecommons.org
       dst: classy.org/give/313412/#!/donation/checkout
+    - crt: redirects.creativecommons.org
+      src: e-donate.creativecommons.org
+      dst: 'https://www.classy.org/campaign/20th-anniversary-better-sharing-campaign/c338989?c_src=email&c_src2=signature'
```

Comment:
@TimidRobot the inclusion of the preceding `https://` is not present on any of other entries. Will that matter?